scl/elasticsearch/plugin.conf

```diff
@@ -68,6 +68,14 @@ block destination elasticsearch2(
   concurrent_requests("1")
   skip_cluster_health_check("")
   cluster_url("")
+  ssl_keystore_filepath("")
+  ssl_keystore_password("")
+  ssl_truststore_filepath("")
+  ssl_truststore_password("")
+  ssl_insecure("false")
+  http_auth_type("none")
+  http_auth_type_basic_username("")
+  http_auth_type_basic_password("")
```

Comment:
I think at the very least the SCL wrapper should use option names that are similar to how other destinations get SSL related parameters. Stuff like ca-file() & ca-dir() to specify the trusted CA certificates (ca-file() as a text file containing many CA certs in PEM format, ca-dir() to specify a directory with hashed filenames that point to CA directories).

Also, keystore() is I guess used for private keys, where we use: key-file() for the private key and cert-file() for the certificate. Is it possible to map arguments this way?